src/vs/workbench/contrib/comments/browser/commentsAccessibleView.ts

```diff
@@ -84,3 +112,41 @@ class CommentsAccessibleContentProvider extends Disposable implements IAccessibl
 		return this.provideContent();
 	}
 }
+
+class CommentsThreadWidgetAccessibleContentProvider extends Disposable implements IAccessibleViewContentProvider {
+	readonly id = AccessibleViewProviderId.CommentThread;
+	readonly verbositySettingKey = AccessibilityVerbositySettingId.Comments;
+	readonly options = { type: AccessibleViewType.View };
+	constructor(@ICommentService private readonly _commentService: ICommentService,
+		@IEditorService private readonly _editorService: IEditorService,
+		@IUriIdentityService private readonly _uriIdentityService: IUriIdentityService,
+	) {
+		super();
+	}
+	// TODO: actions
```

Comment:
unclear how to do this, any suggestions @alexr00?

Reply:
That scoped context key service is not accessible right now. I think the easiest way to get at is to grab the `CommentsController` from here: 

https://github.com/microsoft/vscode/blob/5da1e4ae706213256ab61b7b4fc21a1b8c9a4c19/src/vs/workbench/contrib/comments/browser/commentsController.ts#L685-L687

Then, on [`CommentController`](https://github.com/microsoft/vscode/blob/5da1e4ae706213256ab61b7b4fc21a1b8c9a4c19/src/vs/workbench/contrib/comments/browser/commentsController.ts#L422-L423), expose a `getThreadContext`, which would find the thread in [`_commentThreadWidgets`](https://github.com/microsoft/vscode/blob/5da1e4ae706213256ab61b7b4fc21a1b8c9a4c19/src/vs/workbench/contrib/comments/browser/commentsController.ts#L426-L427) and return that [`ReviewZoneWidget`s](https://github.com/microsoft/vscode/blob/5da1e4ae706213256ab61b7b4fc21a1b8c9a4c19/src/vs/workbench/contrib/comments/browser/commentThreadZoneWidget.ts#L102-L103) context key service. `ReviewZoneWidget` would also need to expose a `getContext`.

Reply:
decided against actions here, see below comment